docs/shared/monorepo-ci-github-actions.md

```diff
@@ -30,19 +30,19 @@ jobs:
       - run: npx nx affected --target=build --parallel=3
 ```
 
-The `pr` and `main` jobs implement the CI workflow. Setting `timeout-minutes` is needed only if you have very slow tasks.
+The `pr` and `main` jobs implement the CI workflow. Setting the `timeout-minutes` is needed only if you have very slow tasks.
```

Comment:
```suggestion
The `pr` and `main` jobs implement the CI workflow. Setting the `timeout-minutes` property is needed only if you have very slow tasks.
```

Reply:
Went with "Setting timeout-minutes" as it's shorter